patches/arcade/0015-remove-netcoreapp2.0-targets.patch

```diff
@@ -0,0 +1,65 @@
+From 72434eb5c25119ae74cbf2451187c0285a968ebf Mon Sep 17 00:00:00 2001
+From: Tom Deseyn <tom.deseyn@gmail.com>
+Date: Tue, 17 Nov 2020 14:01:25 +0100
+Subject: [PATCH] arcade: remove netcoreapp2.0 targets
```

Comment:
I think this will need to remove everything but `net5.0` to work with the NuGet `net5.0`-only build?